TelegramManager/ui/widgets/session_form.py

```
@@ -55,3 +71,59 @@ def _iniciar_autenticacao(self) -> None:
+        try:
+            estrutura = json.loads(payload)
+        except json.JSONDecodeError:
+            dados = {}
+            for parte in re.split(r"[;\n]", payload):
+                if "=" not in parte:
+                    continue
+                chave, valor = parte.split("=", 1)
+                dados[chave.strip().lower()] = valor.strip()
+        else:
+            if not isinstance(estrutura, dict):
+                raise ValueError("O QR Code precisa conter um objeto JSON ou pares chave=valor.")
```

Ruff (0.14.1)

Prefer `TypeError` exception for invalid type

(TRY004)